rs/tests/testing_verification/testnets/small_high_perf.rs

```diff
@@ -66,21 +71,74 @@ pub fn setup(env: TestEnv) {
             memory_kibibytes: Some(AmountOfMemoryKiB::new(480 << 20)),
             boot_image_minimal_size_gibibytes: Some(ImageSizeGiB::new(2_000)),
         })
+        .use_specified_ids_allocation_range()
         .add_subnet(Subnet::new(SubnetType::System).add_nodes(1))
         .add_subnet(Subnet::new(SubnetType::Application).add_nodes(1))
         .setup_and_start(&env)
         .expect("Failed to setup IC under test");
-    install_nns_with_customizations_and_check_progress(
-        env.topology_snapshot(),
-        NnsCustomizations::default(),
-    );
+    check_nodes_health(&env);
+    install_nns_canisters_at_ids(&env);
     BoundaryNode::new(String::from(BOUNDARY_NODE_NAME))
         .allocate_vm(&env)
         .expect("Allocation of BoundaryNode failed.")
         .for_ic(&env, "")
         .use_real_certs_and_dns()
         .start(&env)
         .expect("failed to setup BoundaryNode VM");
-    env.sync_with_prometheus();
+    env.sync_with_prometheus(BOUNDARY_NODE_NAME);
     await_boundary_node_healthy(&env, BOUNDARY_NODE_NAME);
 }
+
+fn check_nodes_health(env: &TestEnv) {
```

Comment:
Nitpicking but I would mention the word "await" in the name of the function to indicate this will wait until the nodes are healthy and not nearly check if they're currently healthy.
```suggestion
fn await_nodes_health(env: &TestEnv) {
```

Reply:
done.